pipeline-runner/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN echo ".libPaths(c('$RENV_LIB', .libPaths()))" >> $(R RHOME)/etc/Rprofile.sit
 
 # install renv to install required R packages
 RUN R -q -e "install.packages('remotes', repos = c(CRAN = 'https://cloud.r-project.org'))" && \
-    R -q -e "remotes::install_github('rstudio/renv@0.16.0')" && \
+    R -q -e "remotes::install_github('rstudio/renv')" && \
```

Comment:
you should fix the version to latest here.

Reply:
be mindful that they changed the way they write versions. Now it's like "v1.0.5"

Reply:
Also, you will probably need to update the config files in the .renv directory

Reply:
ok thanks, done